webpack.common.js

```diff
@@ -30,7 +30,7 @@ module.exports = () => {
         APP_ENV: JSON.stringify(APP_ENV),
         API_URL: JSON.stringify(process.env.API_URL),
         APP_VERSION: JSON.stringify(version),
-        LAGO_DISABLE_SIGNUP: process.env.LAGO_DISABLE_SIGNUP,
+        LAGO_DISABLE_SIGNUP: Boolean(process.env.LAGO_DISABLE_SIGNUP),
```

Comment:
Just to be sure: how this will behave if the env variable is not set (null)?

Reply:
`Boolean(null)` returns false

Reply:
Boolean returns false for null, undefined, 0 or empty string

Reply:
Nice!